_component_design/text-input.md

```diff
@@ -337,7 +337,7 @@ Consider the following guidelines when you construct a multi-line text area:
 - Text areas scroll vertically when the cursor reaches the bottom of the field.
 - A text area can include an optional draggable resize icon in the lower corner.
 - All text input specifications apply to a text area.
-- Text area states follow the same layout and behaviors as text inputs.
+- Text area states follow the same layout and behaviors as [text input states]({{site.baseurl}}/components/text-input.html/#states).
```

Comment:
Still need to fix this

```suggestion
- Text area states follow the same layout and behaviors as [text input states]({{site.baseurl}}/components/text-input.html#states).
```

Reply:
What is there to fix? Remove the '/'? I have tried without the / and the anchor doesn't work. It does work with the /.

Reply:
Hm, when I test it, it doesn't work with the slash, but does without. Will look after demo

Reply:
So I think you'll have to trust that we have to remove the slash on this one. It's an issue with the site's backend. For a comparison, try the following:
http://helix.rax.io/style/typography.html#link
vs
http://helix.rax.io/style/typography.html/#link
The first works; the second does not.

Reply:
Ok, will make that change and push! Thanks for researching this one!